components/BiolucidaViewer/BiolucidaViewer.vue

```diff
@@ -1,20 +1,16 @@
 <template>
   <div class="biolucida-viewer">
     <template v-if="data.status !== 'error'">
-      <p>
-        <a :href="data.share_link" target="_blank">
-          <bf-button>
-            View Full Screen
-            <svg-icon
-              class="icon-full-screen"
-              name="icon-full-screen"
-              height="24"
-              width="24"
-            />
-          </bf-button>
-        </a>
-      </p>
-      <iframe ref="biolucida" :src="data.share_link" />
+      <el-row class="mb-2 justify-space-between" type="flex">
+        <bf-button class="ml-8 btn-copy-permalink-solid" @click="launchViewer">
+          View in 3D
+        </bf-button>
+        <bf-button class="" @click="queryView">
+          <svg-icon name="icon-permalink" height="24" width="24" />
```

Comment:
If we want to match the other parts of the site, I would move the permalink to here and have it be icon only

![image](https://user-images.githubusercontent.com/37255664/166835204-dc5ee252-b4b0-40f1-a90f-754faa163f3b.png)

Can see an example of permalinks elsewhere here:
https://sparc.science/news-and-events/events/RifYKQymUFDJymzyyehm5

![image](https://user-images.githubusercontent.com/37255664/166836425-9741a481-b1f0-4c24-9254-93139d972703.png)

Reply:
I have matched the button styling but have not moved it.  I'll let UX decide if it should be moved.